cooltools/api/coverage.py

```diff
@@ -102,24 +114,48 @@ def coverage(
         raise ValueError(
             "Please, specify ignore_diags and/or IC balance this cooler! Cannot access the value used in IC balancing. "
         )
-
+    if clr_weight_name is not None:
```

Comment:
somehow I missed it from the main review:
I wonder if we could streamline this clause in a simial manner as here https://github.com/open2c/cooltools/blob/2ba3c6ef6bb20b182b81f47ee03707298fa101fa/cooltools/api/expected.py#L1110

so it would be like so:
```python
    if clr_weight_name is None:
        # summing raw pixel counts
        pixel_weight_key = "count"
    elif is_cooler_balanced(clr, clr_weight_name):
        # initialize balancing bias and masking
        bias = clr.bins()[clr_weight_name][:]
        bias_na_mask = np.isnan(bias)  # remember masked bins
        bias = np.nan_to_num(bias)
        # summing balanced pixels
        pixel_weight_key = "balanced"
    else:
        raise ValueError(
            "cooler is not balanced, or"
            f"balancing weight {clr_weight_name} is not available in the cooler."
        )

```

Reply:
is the pixel weight always stored in 'balanced' or is it named after the specific weight clr_weight_name?

Reply:
also does it modify the input cooler? if so that would probably not be desired behavior?

Reply:
No, here we define how to call balanced pixel values (kind of like in expected - transform function) and we use that name in the _apply_balancing function, regardless of how balancing weights are called in cooler

Reply:
No no - balanced values don't get stored in the cooler - same as in expected summations - we apply balancing on the fly - via transform - use the values - and that's it - garbage collect takes over it I guess eventually

Reply:
Implemented your suggestion @sergpolly !